src/imu.cpp

```diff
@@ -345,6 +345,17 @@ std::map <std::string, std::string> Imu::Info::toMap() const {
   return map;
 }
 
+uint16_t Imu::Info::getModelNumber() const {
+  // Simply convert the first contiguous group of numbers into an int
+  int modelInt = std::stoi(modelNumber.c_str());
```

Comment:
`std::stoi` takes a `std::string` as the first argument, so no need to use `.c_str()`